web/modules/custom/joinup_paragraphs/joinup_paragraphs.info.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Joinup paragraphs'
+type: module
+description: 'Contains settings, installation config and functionality related to paragraphs needed in Joinup.'
+core: 8.x
+package: Joinup
+
+dependencies:
+  - paragraphs:paragraphs
+
+config_devel:
+  install:
+    - field.field.paragraph.simple_paragraph.field_body
+    - field.storage.paragraph.field_body
+    - paragraphs.paragraphs_type.simple_paragraph
```

Comment:
Moved them to optional. I don't want to remove them since we already didn't reach the point to remove all of them. Just for a temporary consistency. We only recently transitioned to the optional folder, let's not introduce the next step already without even a ticket.